packages/react-native/src/Authenticator/hooks/useFieldValues/types.ts

```diff
@@ -26,10 +28,13 @@ export interface UseFieldValuesParams<FieldType extends TypedField> {
    * machine "SUBMIT"" event handler, validates `field` value against machine validation rules
    */
   handleSubmit: MachineEventHandlers['handleSubmit'];
+
+  validationErrors?: AuthenticatorMachineContext['validationErrors'];
 }
 
 export interface UseFieldValues<FieldType extends TypedField> {
   fields: FieldType[]; // return either radio or text
+  formValidationErrors?: ValidationError;
```

Comment:
What do you think if naming this `fieldValidationErrors`? Thinking it better aligns with the hook name, and also since these are validation errors at the field level